hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiTokenUpdate.java

```diff
@@ -275,6 +285,29 @@ protected Consumer<TransactionBody.Builder> opBodyDef(HapiSpec spec) throws Thro
                                     Timestamp.newBuilder().setSeconds(t).build()));
                             autoRenewPeriod.ifPresent(secs -> b.setAutoRenewPeriod(
                                     Duration.newBuilder().setSeconds(secs).build()));
+                            // We often want to use an existing contract to control the keys of various types (supply,
```

Comment:
Second time I'm seeing this block!  Maybe it should go somewhere else in one place? Like in `TxnFactory`, at first glance?